### PR TITLE
Fixes hivebot swarm cores hitscanning people 4 screens away with a buzzsaw

### PR DESCRIPTION
**Pull request by silicons**
Yes, I very, VERY ded.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
This is unlikely to fix it using the buzzsaw at unexpected ranges.

**Comment by The-clown-known-as-Pepe** [comment]
While i can't confirm this is a fix

I can confirm that kekhi was rekt by said buzzsaw.

**Comment by kevinz000** [comment]
eh atleast it fixes it not losing its targets.

**Comment by kevinz000** [comment]
also what the fuck it kept sawing me for 4 minutes and kept doing so after I reached centcom?!!

File: code/game/machinery/hivebot_swarm_core.dm [hunk 1]
```diff
@@ -68,11 +68,8 @@
 	say("SUBJECT STATUS: THREAT TO SWARM. TARGET LOCKED.")
 
 /obj/machinery/hivebot_swarm_core/proc/defend_the_swarm()
-	var/atom/movable/threat_to_swarm = src.threat_to_swarm
 	var/mob/living/L = threat_to_swarm
-	if(!istype(L) || L.stat)
-		L = null
```
Comment (Cyberboss) [inline code/game/machinery/hivebot_swarm_core.dm]: This was all here for a reason

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Cyberboss** [comment]
I'll fix it, calm your tits

**Pull request closed without merging**